CodeGeneration/Sources/SyntaxSupport/CommonNodes.swift

```diff
@@ -13,34 +13,34 @@
 public let COMMON_NODES: [Node] = [
   // code-block-item-list -> code-block-item code-block-item-list?
   Node(
-    name: "CodeBlockItemList",
+    kind: .codeBlockItemList,
+    base: .syntaxCollection,
     nameForDiagnostics: nil,
-    kind: "SyntaxCollection",
-    element: "CodeBlockItem"
+    elementChoices: [.codeBlockItem]
   ),
 
   // code-block-item = (decl | stmt | expr) ';'?
   Node(
-    name: "CodeBlockItem",
+    kind: .codeBlockItem,
+    base: .syntax,
     nameForDiagnostics: nil,
-    description: "A CodeBlockItem is any Syntax node that appears on its own line inside a CodeBlock.",
-    kind: "Syntax",
+    documentation: "A CodeBlockItem is any Syntax node that appears on its own line inside a CodeBlock.",
```

Comment:
Not part of this, but we could add ticks around `CodeBlockItem` and `CodeBlock` (and properly also many other places) to add doc linking